mutatex/utils.py

```diff
@@ -421,43 +422,59 @@ def get_foldx_sequence(pdb, multimers=True):
 
     residue_list = []
     sequences = {}
+    positions = {}
+
     for model in structure:
         for chain in model:
             chain_name = chain.get_id()
             sequences[chain_name] = ''
+            positions[chain_name] = ''
             for residue in chain:
                 try:
                     res_code = PDB.Polypeptide.three_to_one(residue.get_resname())
+                    pos_code = str(residue.get_id()[1])
                 except:
                     log.warning("Residue %s in file %s couldn't be recognized; it will be skipped" %(residue, pdb))
                     continue
                 if not multimers:
                     residue_list.append(tuple(["%s%s%d" % (res_code, chain.get_id(), residue.get_id()[1])]))
                 else:
                     sequences[chain_name] += res_code
+                    positions[chain_name] += pos_code
 
     if multimers:
-        collated_chains = []
+        collated_chains_pos = []
+        collated_chains_seq = []
         seq_ids, seqs = list(zip(*list(iteritems(sequences))))
+        pos_ids, pos = list(zip(*list(iteritems(positions))))
```

Comment:
here you obtain a list with strings with all residue numbers pasted one after the other, e.g.`14151617` for `(14, 15, 16, 17)`, however this does not correspond to a unique sequence of residue numbers, as it is for the sequence, because numbers are made of more than one character. So for instance sequences

```
11 12 13 14
1 11 21 314
```

result in the same overall "pasted" sequence `11121314`. An alternative idea could be to have a tuple instead of a string sequence, which is unique (a tuple is also hashable as is a string)